core/include/mutex.h

```diff
@@ -55,11 +55,14 @@ typedef struct {
 #define MUTEX_INIT_LOCKED { { MUTEX_LOCKED } }
 
 /**
- * @internal
+ * @cond INTERNAL
```

Comment:
To elaborate on the error: `@internal` made this show up as an undocumented macro in Doxygen. It seems like Doxygen treats `@internal` as if the doc block does not even exist, instead of hiding it.